cmd/node/node.go

```diff
@@ -501,7 +501,7 @@ func (app *SpacemeshApp) initServices(nodeID types.NodeId,
 	if mdb.PersistentData() {
 		trtl = tortoise.NewRecoveredAlgorithm(mdb, app.addLogger(TrtlLogger, lg))
 		msh = mesh.NewRecoveredMesh(mdb, atxdb, app.Config.REWARD, trtl, app.txPool, atxpool, processor, app.addLogger(MeshLogger, lg))
-
+		go msh.CacheWarmUp()
```

Comment:
I'm out of context so I don't know why we need that to be non blocking. so I'm not saying anything about that. though for this purpose I understand why you moved it from inside `NewRecoveredMesh` to here...

Reply:
blocks the startup till warmup is done, since the app waits for the grpc server